package.json

```diff
@@ -7,6 +7,17 @@
   "author": "Nerdbord",
   "license": "MIT",
   "scripts": {
-    "server": "node index.js"
+    "server": "node index.ts",
+    "startDev": "tsnd index.ts",
```

Comment:
**issue:** Same here - if I clone your repo, I won't be able to run it, as it's configured for VSC specifically. `ts-node` will be totally enough for that purpose